packages/manager/core/generator/app/templates/package.json.hbs

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "{{ packageName }}",
+  "version": "0.0.0",
+  "description": "{{ description }}",
+  "private": "true",
```

Comment:
```suggestion
  "private": true,
```

Sinon la commande build à la racine bug

Reply:
The reason is `private` is boolean, not a string.
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#private